irodsadmin/management/commands/irodsorphans.py

```diff
@@ -151,37 +151,34 @@ def get_orphans(irods, irods_backend, expected, assays):
                 irods, irods_backend.get_path(assay)
             ):
                 if collection.path not in expected:
-                    orphans.append(collection.path)
-    return orphans
-
-
-def get_output(orphans, irods_backend, irods):
-    lines = []
-    for orphan in orphans:
-        stats = irods_backend.get_object_stats(irods, orphan)
-        m = re.search(r'/projects/([^/]{2})/(\1[^/]+)', orphan)
-        if m:
-            uuid = m.group(2)
-            try:
-                project = Project.objects.get(sodar_uuid=uuid)
-                title = project.full_title
-            except Project.DoesNotExist:
-                title = '<DELETED>'
-        else:
-            uuid = '<ERROR>'
-            title = '<ERROR>'
-        lines.append(
-            ';'.join(
-                [
-                    uuid,
-                    title,
-                    orphan,
-                    str(stats['file_count']),
-                    filesizeformat(stats['total_size']).replace(u'\xa0', ' '),
-                ]
-            )
+                    log_orphan(collection.path, irods_backend, irods)
+
+
+def log_orphan(orphan, irods_backend, irods):
```

Comment:
These helpers should be refactored as protected methods within the command class.

That way, we can also simply set `self.irods_backend` in `__init__()` and eliminate the need to provide it to each helper function as an argument. The `irods` session object should still be given as arg though, because we want to init it in a `with` block in `handle()` as we currently do.